Makefile

```diff
@@ -317,6 +317,23 @@ format-fix: ## Runs `npm run format-fix` for all projects
 	@echo "==============================================="
 	@cd database && npm run format-fix && cd ..
 
+## ------------------------------------------------------------------------------
+## Run `npm` commands for all projects ./.pipeline
+## ------------------------------------------------------------------------------
+pipeline-install: ## Runs `npm install` for all projects
+	@echo "==============================================="
+	@echo "Running /api/.pipeline install"
+	@echo "==============================================="
+	@cd api/.pipeline && npm install && cd ../..
```

Comment:
Thoughts on using `pushd` and `popd` in lieu of of `cd`?

Reply:
No strong thoughts, wasn't aware of pushd or popd tbh.

Did a quick search, seems like both do more or less the same thing.

I'd probably just leave them as is, unless we have a reason to change them